docs/source/getting-started/installation.rst

```diff
@@ -12,7 +12,7 @@ The SDK currently runs on MacOS or Linux. For Windows support, we recommend inst
 You need to install:
 
 1. `Visual Studio Code <https://code.visualstudio.com/download>`_.
-2. `JDK 8 or greater <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_.
+2. `JDK 8 <https://openjdk.java.net/install/>`_.
```

Comment:
This page does not contain installation instructions for Mac. :(

I followed the instructions at https://github.com/AdoptOpenJDK/homebrew-openjdk#readme but I'm not sure we want to tell this to our users.

Reply:
ugh, okay, this is going to need some more thought, isn't it...